app/scripts/views/settings.js

```diff
@@ -148,6 +149,34 @@ function ($, modal, Cocktail, Session, BaseView, AvatarMixin,
       return self._showAvatar();
     },
 
+    beforeDestroy: function () {
+      $('body.settings').fadeOut(this.FADE_OUT_SETTINGS_MS, function (){
+        $('body').removeClass('settings').show();
+      });
+    },
```

Comment:
I think this change will introduce a flash of content, see gif:
![flasg](https://cloud.githubusercontent.com/assets/128755/10103280/0170028a-6371-11e5-93f2-3560df7f6fb9.gif)

@zaach ideas?